### PR TITLE
[CCE]: Fix ``kube_proxy_mode`` parameter for ``resource/opentelekomcloud_cce_cluster_v3``

### PR DESCRIPTION
**Pull request by artem-lifshits**
## Summary of the Pull Request
After CCE update ``kube_proxy_mode`` can be managed by users through API.
This PR enables this management.

## PR Checklist

* [x] Refers to: #1025 
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

=== RUN   TestAccCCEClusterV3_invalidNetwork
=== PAUSE TestAccCCEClusterV3_invalidNetwork
=== CONT  TestAccCCEClusterV3_invalidNetwork
--- PASS: TestAccCCEClusterV3_invalidNetwork (30.37s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3_importBasic
=== PAUSE TestAccCCEClusterV3_importBasic
=== CONT  TestAccCCEClusterV3_importBasic
--- PASS: TestAccCCEClusterV3_importBasic (469.85s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3_proxyAuth
=== PAUSE TestAccCCEClusterV3_proxyAuth
=== CONT  TestAccCCEClusterV3_proxyAuth
--- PASS: TestAccCCEClusterV3_proxyAuth (530.74s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3_basic
=== PAUSE TestAccCCEClusterV3_basic
=== CONT  TestAccCCEClusterV3_basic
--- PASS: TestAccCCEClusterV3_basic (498.87s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3_timeout
=== PAUSE TestAccCCEClusterV3_timeout
=== CONT  TestAccCCEClusterV3_timeout
--- PASS: TestAccCCEClusterV3_timeout (501.57s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3NoAddons
=== PAUSE TestAccCCEClusterV3NoAddons
=== CONT  TestAccCCEClusterV3NoAddons
--- PASS: TestAccCCEClusterV3NoAddons (487.48s)
PASS

Process finished with the exit code 0

=== RUN   TestAccCCEClusterV3_withVersionDiff
=== PAUSE TestAccCCEClusterV3_withVersionDiff
=== CONT  TestAccCCEClusterV3_withVersionDiff
--- PASS: TestAccCCEClusterV3_withVersionDiff (470.68s)
PASS

Process finished with the exit code 0
```

### CODE REVIEW

File: opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go [hunk 1]
```diff
@@ -197,9 +197,13 @@ func ResourceCCEClusterV3() *schema.Resource {
...
 				ForceNew: true,
 			},
-			"kube_proxy_mode": { // can't be set via API currently
+			"kube_proxy_mode": {
 				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
 				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"ipvs", "iptables"}, true),
```
Comment (outcatcher) [inline opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:206]: It's better to be split in lines like 
```go
validation.StringInSlice(
	[]string{"ipvs", "iptables"}, true,
),
```

That makes it bit more readable

**Review by Aloento — APPROVED** [review]

File: opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go (hunk 1 above)
Comment (artem-lifshits) [reply opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:206]: @outcatcher it gets auto-formatted to what you see above when you try to do that :)

**Review by anton-sidelnikov — APPROVED** [review]

File: opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go (hunk 1 above)
Comment (outcatcher) [reply opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:206]: @artem-lifshits 
There are examples of working option:
```go
				ValidateFunc: validation.StringInSlice([]string{
					"mongos", "shard", "config", "replica",
				}, true),
```

Comment (artem-lifshits) [reply opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:206]: Ah I see, so the part after the slice would be put on the next line?

Comment (outcatcher) [reply opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go:206]: The slice opening bracket on previous live and closing - on the next 🙃

**Pull request merged**